src/test/java/jsUtils.js

```diff
@@ -165,3 +165,27 @@ function generateUUID(){
     var uuid = '' + java.util.UUID.randomUUID();
     return uuid
 }
+
+/** To validate expected result and actual result with dynamic value
+ * response: will get after execution
+ * constant: constant value in constant yaml file
+ */
+function toGetDynamicValueFromResponse(response, constant)
+{        
+        var responseArray = response.split("  ", 3);   
```

Comment:
If this is a generic function then how can we determine the split limit is always `3` ?  I think this would be more relevant if this value can be dynamic instead of hard coded.